### DISCUSSION TITLE
Ignoring all-day events

### DISCUSSION THREAD

**Discussion by califa** [YukiGasai/obsidian-google-calendar#175] [General]
Hey I might have missed something obvious in the settings or docs, but is there a way to ignore all the all-day events in my calendar?

**Comment by mhafellner** [comment]
I was also looking for this and found it by pure luck while thinking "let's try this random action".

In the GCal Schedule View -> left click on the icon -> a menu opens with an option "settings" (magic moment) -> click settings -> toggle the "show all day events" toggle -> tada
